### ISSUE TITLE
Highlighting all occurrences of selected word (like Notepad++)

### ISSUE THREAD

**Issue by jad986** [serge-rider/dbeaver#222]
Thanks you for your work. 

Highlighting all occurrences of selected word -> I love this feature when I am writing or 
analyzing SQL queries. It will be great if this feature in DBeaver works as in Notepad++.

**Comment by carlosjrcabello** [comment]
Hi @jad986,

It's so interesting and will aggregate a lot to the DBeaver. I don't have idea about how complex it is to develop, but, for future features it will be welcome.

**Comment by serge-rider** [comment]
Agreed, it'd be a great feature. Unfortunately it won't be easy, Eclipse text editor doesn't support this feature out of the box. Although JDT does so I could try to make it by example..

**Comment by giquieu** [comment]
Hello,
This functionality is very interesting! A feature often used in SQL Editor :)
Is there any news related?

Hugs!

**Comment by serge-rider** [comment]
Maybe Glance (http://ystrot.github.io/glance/) could be helpful here..

**Comment by adamitj** [comment]
Glance should work, indeed. Any ideas on how to install this plugin in DBeaver? Is there any "backdoor" for testing purposes?

**Comment by serge-rider** [comment]
In 3.7.7 EE you can install it as in Eclipse: main menu Help->Install new software.
Glance update URL is http://www.exyte.com/glance/updates
Glance shortcuts conflicts with DBeaver. You'll need to redefine CTRL+ALT+F to some other sequence (in preferences).

**Comment by icaptnbob** [comment]
Eclipse provides this option for Java editors, using the "Mark Occurrences" option. Perhaps this can be extended to SQL Editors.

See this question for the details and process for Java editors here: 
https://stackoverflow.com/questions/10310597/cannot-highlight-all-occurrences-of-a-selected-word-in-eclipse

The "Occurrences" doesn't appear in DBeaver in either Text or SQL editors.

**Comment by Xylandor** [comment]
How do you disable this feature in v5.2.4? There is no **SQL Editor** node in preferences under **General**. I even tried searching for the word "mark", but nothing shows. Here's my pref window:

![2018-11-06 13_51_31-preferences](https://user-images.githubusercontent.com/24617679/48089990-5c6c1d00-e1cb-11e8-8fbd-52ac6ac3e6d8.png)

**Comment by serge-rider** [comment]
In 5.2.3 and 5.2.4 all database editors prefs were moved to Database->Editors:
![image](https://user-images.githubusercontent.com/6398845/48090119-20f74000-e217-11e8-9fbe-e6e78d038e1e.png)

**Comment by Shantanu101998** [comment]
Hello! Marking all occurrences is a good step in the direction! But can we go a step further and "select" all occurrences. Like the shortcut "Command + D" in VS code. I know there is always an option of "Find and replace all" but it is nowhere as easy to use as the above mentioned option. Thanks!
